gm4_storage_drawers/data/storage_drawers/functions/main.mcfunction

```diff
@@ -0,0 +1 @@
+function storage_drawers:all/detection_master_main
```

Comment:
This is a pointless function call, instead put all the commands of `storage_drawers:all/detection_master_main` inside this function.